_posts/2021-01-13-hapi-fhir.md

```diff
@@ -379,9 +399,9 @@ public class RestfulServerWithCors extends RestfulServer {
 }
 ```
 
-#### Authorization Interceptor
+#### AuthorizationInterceptor
 
-HAPI FHIR provides you with an Authorization interceptor which can be helpful with the task of determining whether a user has the appropriate permission to perform a given task on a FHIR server. 
+HAPI FHIR provides you with an AuthorizationInterceptor which can be helpful with the task of determining whether a user has the appropriate permission to perform a given task on a FHIR server. 
```

Comment:
you've got "task" twice in the same sentence
suggested rephrase: "... which can be helpful to determine whether ..."